frontend/scss/components/_grid.scss

```diff
@@ -56,7 +56,7 @@
 .row {
   margin-left: auto;
   margin-right: auto;
-  margin-bottom: 20px;
```

Comment:
This same component can be selected and the style added on '_partners.scss', this way we avoid changing core components that affect multiple pages.